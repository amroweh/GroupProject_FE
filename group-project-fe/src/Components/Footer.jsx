import React from 'react'
import { NavLink } from 'react-router-dom'
import '../Components/css/footer.css'

const Footer = () => {
  return (
    <footer>
      {/* Bootstrap container to adjust width */}
      <div className='container'>
        {/* Left Footer Section */}
        <div id='footer-logo-section'>
          <img src="/sky_logo.png" alt=""/>
          <span>Â© 2023 Sky UK</span>
        </div>
        {/* Middle Footer Section */}
        <ul id='footer-middle-section'>
          <li><NavLink className="footer-nav-link" to="/">Home</NavLink></li> 
          <li><NavLink className="footer-nav-link" to="/">About</NavLink></li> 
          <li><NavLink className="footer-nav-link" to="/">Movies</NavLink></li> 
          <li><NavLink className="footer-nav-link" to="/">Sky</NavLink></li> 
        </ul>
        {/* Right Footer Section */}
        <div id='footer-language-section'>
          <label htmlFor="footer-countries-dropdown">Country: </label>
          <select name="footer-countries-dropdown" id="footer-countries-dropdown">
            <option value="uk">UK</option>
            <option value="ireland">Ireland</option>
          </select>
        </div>     
      </div>  
    </footer>
  )
}

export default Footer