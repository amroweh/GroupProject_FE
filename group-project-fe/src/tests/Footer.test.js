import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '../Components/Footer'

describe("Footer matches snapshot ", ()=>{
    test("footer snapshot test", ()=>{
        const footer = create(<Footer />)
        expect(footer.toJSON()).toMatchSnapshot();
    })
})