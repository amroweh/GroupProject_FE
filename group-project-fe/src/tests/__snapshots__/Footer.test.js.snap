// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot   1`] = `
<footer>
  <div
    className="container"
  >
    <div
      id="footer-logo-section"
    >
      <img
        alt=""
        src="/sky_logo.png"
      />
      <span>
        Â© 2023 Sky UK
      </span>
    </div>
    <ul
      id="footer-middle-section"
    />
    <div
      id="footer-language-section"
    >
      <label
        htmlFor="footer-countries-dropdown"
      >
        Country: 
      </label>
      <select
        id="footer-countries-dropdown"
        name="footer-countries-dropdown"
      >
        <option
          value="uk"
        >
          UK
        </option>
        <option
          value="ireland"
        >
          Ireland
        </option>
      </select>
    </div>
  </div>
</footer>
`;
